eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -104,35 +109,6 @@ jobs:
               artifactName: 'artifacts' 
               targetPath: $(Build.ArtifactStagingDirectory)
 
-  - job: 'Test_Linux_Python38'
```

Comment:
Do we think we need to have a "nightly" for the next version past 3.8 to try and keep up with the latest versions?

Reply:
As discussed, we have an edge python pipeline to test next version of python.